bigquery_etl/publish_json.py

```diff
@@ -12,7 +12,9 @@
 METADATA_FILE = "metadata.yaml"
 SUBMISSION_DATE_RE = re.compile(r"^submission_date:DATE:(\d\d\d\d-\d\d-\d\d)$")
 QUERY_FILE_RE = re.compile(r"^.*/([a-zA-Z0-9_]+)/([a-zA-Z0-9_]+)_(v[0-9]+)/query\.sql$")
-
+MAX_JSON_SIZE = 1 * 1024 * 1024 * 1024  # 1 GB as max. size of exported JSON files
+# maximum number of JSON output files, output file names are only up to 12 characters
+MAX_FILE_COUNT = 999_999_999_999
```

Comment:
Thinking about it, should we make this smaller? If we hit 10k files something wrong has probably happened and we should bail.